packages/govtnz-ds-website/src/commons/Example.tsx

```diff
@@ -321,7 +321,7 @@ export default class Example extends Component<Props, State> {
               />
             </div>
             <Details className="example__details" onChange={this.clickFormat}>
-              <Summary className="example__summary">
+              <Summary id={summaryId} className="example__summary">
```

Comment:
@holloway I've added the `id` prop here but VS Code still complains that `id` is missing the the declaration. Is there anything I need to do in order to put that in the declaration?